persistence/src/commonMain/db_user/com/wire/kalium/persistence/ConversationFolders.sq

```diff
@@ -17,6 +17,19 @@ CREATE TABLE LabeledConversation (
       PRIMARY KEY (folder_id, conversation_id)
 );
 
+getFolderWithConversations:
```

Comment:
question: is it supposed to return all folders? Then maybe name it `getAllFoldersWithConversations`? 🤔